### DISCUSSION TITLE
mafTools Plots - Not Catchable?

### DISCUSSION THREAD

**Discussion by HHayman** [PoisonAlien/maftools#820] [General]
Heylo

I've noticed that (off the top of my head, all actually) most mafTools outputs can't be assigned to an object. 

For example, I can't do; 

```r
lolpop <- lollipopPlot(
        maf = maf,
        gene = gene,
        AACol = "Transcript_Annotation",
        showMutationRate = TRUE
      )
```

lolpop is null.

Usually I'm saving plots to tiffs, so I haven't really noticed an issue. However I'm working in R Markdown just now and I need to be able assign plots to objects for arrangements etc, as one might with say ggplot. 

As far as I can tell, mafTools plots are never actually returned? I wondered if this is by design to prevent some issue? Is there a way to get around this?

I opted for a discussion rather than an issue since it doesn't seem that it isn't working, just that it isn't designed to function this way.

**Comment by PoisonAlien** [comment]
Hi,
All the plots generated by maftools are in base graphics. Unfortunately, plots can not be saved as an object.
